fleet_adapter_mir/fleet_adapter_mir/MiRCommandHandle.py

```diff
@@ -504,7 +504,7 @@ def dock_closure():
                 # Docking completed
                 if not self.dry_run:
                     if (self.rmf_docking_executed
-                            and api_response['state_id'] == MiRState.READY):
+                        and (api_response['state_id'] == MiRState.READY)):
```

Comment:
Undo style change.

Reply:
Undid in https://github.com/RBinsonB/fleet_adapter_mir/commit/3d3304ea793e85bd536b79644604653f3c557848